src/specific/s_clock.c

```diff
@@ -2,7 +2,7 @@
 
 #include "global/vars.h"
 
-#include <windows.h>
+#include <time.h>
```

Comment:
This needs to stay as <windows.h> because we use its high precision counters here (used to maintain desired FPS).

Reply:
Ya I moved it to the header when I added that Windows time function that's now gone. Good catch.

Reply:
And we don't need `<time.h>`.